modules/compatibility-tag/base-module.php

```diff
@@ -0,0 +1,143 @@
+<?php
+namespace Elementor\Modules\CompatibilityTag;
+
+use Elementor\Core\Utils\Version;
+use Elementor\Core\Utils\Collection;
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+abstract class Base_Module extends BaseModule {
+	/**
+	 * @var Compatibility_Tag
+	 */
+	private $compatibility_tag_service;
+
+	/**
+	 * @return string
+	 */
+	public function get_name() {
+		return 'compatibility-tag';
+	}
+
+	/**
+	 * Return all the plugins
+	 *
+	 * This protected method created for tests reasons.
+	 *
+	 * @return Collection
+	 */
+	protected function get_plugins() {
+		static $plugins;
+
+		if ( ! $plugins ) {
+			$plugins = Collection::create( get_plugins() );
```

Comment:
Why don't you use `new Collection()`?

Reply:
We can also use `new Collection()`. the `create` method is an aesthetic choice inspired by the `Laravel` syntax. its allow us to make fluent chain of methods without extra parenthesis:
```php
Collection::create( [] )
  ->map(...)
  ->filter(...)
  ->all()
```

vs

```php
( new Collection( [] ) )
  ->map(...)
  ->filter(...)
  ->all()
```